espnet/asr/asr_utils.py

```diff
@@ -576,3 +577,61 @@ def plot_spectrogram(plt, spec, mode='db', fs=None, frame_shift=None,
                     labelbottom=labelbottom, labelleft=labelleft,
                     labelright=labelright, labeltop=labeltop)
     plt.axis('auto')
+
+
+class TransformDataset(torch.utils.data.Dataset):
```

Comment:
Because this class can be also useful for TTS, ST and MT, can you put this to new module like `espnet.utils.dataset`?

Reply:
I agree. It's better to put this in a new module.